includes/kp-functions.php

```diff
@@ -272,7 +272,7 @@ function kp_is_order_pay_page() {
  * @return bool
  */
 function kp_is_wc_blocks_order( $order ) {
-	return $order && $order->is_created_via( 'store-api' );
+	return $order && method_exists( $order, 'is_created_via' ) && $order->is_created_via( 'store-api' );
```

Comment:
I think this will generate another error with PHP 8.1+, since i believe the error with calling is_created_via happens when the order is false or null. method_exists param 1 can only be a string or a object, not a bool or null value. A better fix would be to change this to check if $order is a instance of WC_Order like this:
`return $order && is_a( $order, WC_Order::class ) && $order->is_created_via( 'store-api' );`